MojioSDK/Models/Core/Activity.swift

```diff
@@ -284,6 +286,24 @@ public enum TimelineActivityType: String, Codable {
     case oilLevelWarning = "OilLevelWarning"
     case oilPressureWarning = "OilPressureWarning"
     case oilTemperatureWarning = "OilTemperatureWarning"
+    case airMaintenance = "AirFilterPredictiveMaintenance"
+    case batteryMaintenance = "BatteryPredictiveMaintenance"
+    case unknown
+    
+    public init(from decoder: Decoder) throws {
+        
+        let label = try decoder.singleValueContainer().decode(String.self)
+        
+        if label == "OilLevel" {
```

Comment:
@stepanrazin  Please leave a comment in the code for reason of this changes.

Reply:
@stepanrazin  I would clarify in platform team what name for OilLevel is right. It could be a wrong name "oilLevelWarning" and now they fixed it.